maintainers/maintainer-list.nix

```diff
@@ -7848,6 +7848,12 @@
     githubId = 2845239;
     name = "Tim Put";
   };
+  timstott = {
+    email = "stott.timothy@gmail.com";
+    github = "timstott";
+    githubId = 1334474;
+    name = "Timothy Stott";
+  };
```

Comment:
this change should be in separate commit.

Reply:
Excellent point. I've extracted it to its own commit https://github.com/NixOS/nixpkgs/pull/89183/commits/e536b9e208955af925c8de8ddccc9f6edab0981b